sw/device/tests/usbdev_test.c

```diff
@@ -124,10 +124,10 @@ bool test_main(void) {
   base_printf("\r\n");
   for (int i = 0; i < kExpectedUsbCharsRecved; i++) {
     CHECK(buffer[i] == kExpectedUsbRecved[i],
-          "Recieved char #%d mismatched: exp = %x, actual = %x", i,
+          "Received char #%d mismatched: exp = %x, actual = %x", i,
```

Comment:
Same note as above, I think the USB code changes should be put in a separate PR.